### ISSUE TITLE
Crash when clicking on icon to open

### ISSUE THREAD

**Issue by jacob-swanson** [Helfima/helmod#68]
When I click the button to open Helmod, it crashes with the following notification.

```
Error while running event helmod::on_gui_click (ID 1)
LuaGuiElement API call when LuaGuiElement was invalid.
stack traceback:
	__helmod__/controller/Controller.lua:101: in function 'on_gui_click'
	__helmod__/controller/PlayerController.lua:95: in function 'on_gui_click'
	__helmod__/helmod.lua:154: in function 'on_gui_click'
	__helmod__/control.lua:32: in function <__helmod__/control.lua:31>
```
It worked after I first installed it, but crashes every time now even after uninstalling and reinstalling.

Factorio version: 0.15.16

**Comment by ariscop** [comment]
Same issue when adding to an existing save, starting a save with helmod has worked though

**Comment by Helfima** [comment]
I do not reproduce the problem
if you can push the saved game

**Comment by ariscop** [comment]
[oh_point_15.zip](https://github.com/Helfima/helmod/files/1038264/oh_point_15.zip)

**Comment by Helfima** [comment]
ok I have try it and I have the problem but I work at this time
I look tonight
You can use shortcut [U] to open/close the panel

**Comment by Helfima** [comment]
resolved in 0.5.3

**Comment by Helfima** [comment]
need to click twice time on icon
